check_coverage.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+coverage_threshold=100
+coverage_report=$(tail -n 1 coverage/.last_run.json)
+coverage_percent=$(echo $coverage_report | jq -r '.result.covered_percent')
+
+if [ "$coverage_percent" -lt "$coverage_threshold" ]; then
+  echo "Code coverage is below $coverage_threshold% ($coverage_percent%)"
+  exit 1
+fi
```

Comment:
I think there is a way to config this on ruby side without needing a script. Not sure if the snippet below works 🤔 
```
SimpleCov.at_exit do
  SimpleCov.result.format!
  exit(1) if SimpleCov.result.covered_percent < 90
end
```
https://github.com/simplecov-ruby/simplecov/issues/96#issuecomment-5233083